libraries/AP_DDS/AP_DDS_Topic_Table.h

```diff
@@ -2,6 +2,7 @@
 #include "sensor_msgs/msg/NavSatFix.h"
 #include "tf2_msgs/msg/TFMessage.h"
 
+#include "generated/BatteryState.h"
```

Comment:
Wrong header guard with the new file reorg

Reply:
yep, was fixing it rn. Tested it this time, its working. Sorry about that